src/implementations/vanilla/components/file/file-print.scss

```diff
@@ -85,27 +94,27 @@ $_image-width: 13.125rem;
 .ecl-file__image {
```

Comment:
on print the image margin seems to be on the wrong side
![image](https://github.com/ec-europa/europa-component-library/assets/25579221/3761dbb3-7969-4042-9315-03555cc0ceb6)